model/histogram/float_histogram.go

```diff
@@ -235,7 +235,7 @@ func (r *floatBucketIterator) Next() bool {
 		r.currIdx += span.Offset
 	}
 
-	r.currCount = r.buckets[r.bucketsIdx]
+	r.currCount += r.buckets[r.bucketsIdx]
```

Comment:
Buckets for the FloatHistogram are absolute, not delta encoded.

Reply:
See https://github.com/prometheus/prometheus/blob/4ce01e97707d04c57ab2a39a34400ad64956ff38/model/histogram/float_histogram.go#L23

Reply:
This was an intentional choice to make things easier for the PromQL engine.